sys/drv/bcm2835_gpio.c

```diff
@@ -21,9 +26,21 @@ static void delay(int64_t count) {
 }
 
 #define NGPIO_PIN 54
+#define FDT_PINS_INVAL (-1)
+
+typedef uint32_t gpio_function_t;
```

Comment:
Perhaps it'd be good idea to make valid enums from `gpio_function_t` and `gpio_pull_t`?

Reply:
I removed these. I think that in the future we will want to use defines, because they can be included and reused in DTS files.